client/src/App.js

```diff
@@ -1,17 +1,57 @@
 import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
-
-import { theme } from "./themes/theme";
-import Home from "./pages/Home";
+import { createMuiTheme, responsiveFontSizes } from '@material-ui/core/styles';
+import { BrowserRouter, Route, Redirect } from "react-router-dom";
+import Login from "./pages/Login"
+import Signup from "./pages/Signup"
 
 import "./App.css";
 
+let theme = createMuiTheme({
+  palette: {
+    primary: {
+      light: '#4791db',
+      main: '#3a8dff',
+      dark: '#115293',
+    },
+    secondary: {
+      light: '#ffd95a',
+      main: '#f9a825',
+      dark: '#c17900',
+      contrastText: '#212121',
+    },
+    background: {
+      default: '#ffffff',
+    },
+  },
+  typography: {
+    useNextVariants: true,
+    button: {
+      textTransform: 'none'
+    }
+  },
+  underline: {
+    color: 'red',
+    '&:before': {
+      border: '2px solid red'
+    }
+  }
+});
+
+theme = responsiveFontSizes(theme);
```

Comment:
I feel like you should keep the theme in the theme.js file, and you can modify it there. Good job on making changes to the theme though instead of copying CSS here and there